### PR TITLE
Even More Nouns

### PR DESCRIPTION
**Pull request by MaciejFed**

### CODE REVIEW

File: src/repository/exercises-repository.ts [hunk 1]
```diff
@@ -4391,6 +4384,681 @@ export const db: Schema = {
...
+    {
+      english: 'hotel',
+      portuguese: {
+        word: 'hotel',
+        plural: 'hotéis',
+        gender: 'masculine'
+      }
+    },
+    {
+      english: 'bus',
```
Comment (cleonor) [inline src/repository/exercises-repository.ts:4469]: BRAZILIAN ALERT

File: src/repository/exercises-repository.ts [hunk 2]
```diff
@@ -4391,6 +4384,681 @@ export const db: Schema = {
...
+    {
+      english: 'heart',
+      portuguese: {
+        word: 'coração',
+        plural: 'corações',
+        gender: 'masculine'
+      }
+    },
+    {
+      english: 'square',
```
Comment (cleonor) [inline src/repository/exercises-repository.ts:4615]: more specific

File: src/repository/exercises-repository.ts [hunk 3]
```diff
@@ -6005,17 +6673,9 @@ export const db: Schema = {
       portuguese: 'Isso não costuma ocorrer'
     },
     {
-      english: 'Posso fornecer informações',
+      english: 'Can I provide information?',
       portuguese: 'Posso fornecer informações'
```
Comment (cleonor) [inline src/repository/exercises-repository.ts:6677]: question mark

**Review by cleonor — CHANGES_REQUESTED** [review]

File: src/repository/exercises-repository.ts (hunk 3 above)
Comment (MaciejFed) [reply src/repository/exercises-repository.ts:6677]: fixed

File: src/repository/exercises-repository.ts (hunk 2 above)
Comment (MaciejFed) [reply src/repository/exercises-repository.ts:4615]: fixed

File: src/repository/exercises-repository.ts (hunk 1 above)
Comment (MaciejFed) [reply src/repository/exercises-repository.ts:4469]: fixed

**Pull request closed without merging**